### ISSUE TITLE
Scroll too fast on macOS

### ISSUE THREAD

**Issue by vbocquel** [baktash93/ng2Scrollster#2]
Hi again :smile: 

It seems that scroll is unusable on macOS (Safari & Chrome at least), scroll is too fast.
I believe this has something to do with the `wheelEvent` which values are not the same depending on the OS and/or browser.

**Comment by baktash93** [comment]
Well, I've tested it with Chrome, but not Safari yet. Did you update your package after I made the changes?

**Comment by vbocquel** [comment]
Yes I did (1.2.2), and previous issue was fixed. Thank you for that and your quick answer by the way :wink:

**Comment by vbocquel** [comment]
For your information, this is happening on Chrome / Safari / Firefox on MacOS X 10.11.6 (El Captain). Cannot test on another setup sorry :confused: 
Otherwise it works fine on linux, any browser.

**Comment by baktash93** [comment]
@vbocquel, it seems that the mentioned browsers have a different `event` object than that of the browsers on Windows/Linux. I'll try to make it compatible for them as well. ;)

**Comment by vbocquel** [comment]
Hi :smile: 
Do you need anything to speed up things ? PR, help to test on macOS, ... ?
